### DISCUSSION TITLE
product card > allow for image change on hover (display 2ndary image)

### DISCUSSION THREAD

**Discussion by william-johnson-bloomreach** [bloomreach/limitless-ui-react#38] [Ideas]
For product cards
add a "hover" state, where a 2ndary image is displayed
See example here: https://www.hermes.com/fr/fr/search/?s=carr%C3%A9

**Comment by joerideg** [comment]
Hi William,

Thanks for opening the post and for the explanation of what you're looking to achieve. 

The goal of Limitless UI is to provide a set of flexible, composable UI components that can be used to build e-commerce experiences. Specific customizations to the product grid, like alternating images, would be implemented by you in your application code that utilizes the Limitless UI components.

I'd recommend checking out our [Storybook](https://bloomreach.github.io/limitless-ui-react/) documentation which has live examples and code snippets showing how to use each of the components.

This particular case could be achieved by adding and 'onHover' event handler to the ProductCard.Root or Image that would alter the url of the image passed in to be displayed to a different one.
